js/userCreds.js

```diff
@@ -0,0 +1,63 @@
+;(async function replaceUserCreds() {
+  function traverseNodeChildren(node, callback) {
+    if (!node.childNodes.length) {
+      callback(node)
+    }
+
+    for (const child of node.childNodes) {
+      traverseNodeChildren(child, callback)
+    }
+  }
+
+  function replaceTextContent(node, replacers) {
+    const isTextNode = n => n.nodeType === Node.TEXT_NODE
+
+    traverseNodeChildren(node, (child) => {
+      if (isTextNode(child)) {
+        child.textContent = replacers.reduce(
+          (text, replace) => replace(text),
+          child.textContent
+        )
+      }
+    })
+  }
+
+  function updateUserCreds(username, apiKey = 'YOUR_API_KEY') {
+    const codeExampleNodes = document.querySelectorAll(
+      '[ng-controller=GettingStartedControl]'
```

Comment:
:bulb: Any chance there's an ID or class we could hook into here? Using an Angular controller attribute as our entry point feels like it might be subject to change, but if nothing else is available, no worries.

Reply:
Oooh, good point. Turns out we're using `'[class*=gs-code-container]'` in only the places this controller is referenced. Updated.